pkgs/top-level/python-packages.nix

```diff
@@ -7574,6 +7574,21 @@ let
     propagatedBuildInputs = with self; [ eventlib application gnutls ];
   };
 
+  multipledispatch = buildPythonPackage rec {
+    name = "multipledispatch-${version}";
+    version = "0.4.8";
+   
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/multipledispatch/${name}.tar.gz";
+      md5 = "f0eb0c527cdbe788ca7dcd1511bc49a9";
```

Comment:
Despite there are quite a few packages using md5 sums I think it is better to use sha256.